sunflower/plugins/file_list/file_list.py

```diff
@@ -1587,6 +1587,19 @@ def _edit_selected(self, widget=None, data=None):
 		"""Abstract method to edit currently selected item"""
 		selection_list = self._get_selection_list(relative=False, files_only=True)
 
+		# display error message if selection has no files
+		if selection_list is None:
+			dialog = Gtk.MessageDialog(
+									widget,
+									Gtk.DialogFlags.DESTROY_WITH_PARENT,
+									Gtk.MessageType.INFO,
+									Gtk.ButtonsType.OK,
+									_('No files selected.')
+								)
+			dialog.run()
+			dialog.destroy()
+			return False
```

Comment:
This is event handler method, so it should always return `True` so that default handlers don't pick it up.

Reply:
Sorry, I'm still new to GTK. Pushed a fix.

Reply:
It's okay, your contributions are much appreciated.